astropy/coordinates/tests/test_solar_system.py

```diff
@@ -110,7 +111,9 @@ class TestPositionsGeocentric:
     def setup(self):
         self.t = Time('1980-03-25 00:00')
         self.apparent_frame = TETE(obstime=self.t)
-        # Results returned by JPL Horizons web interface
+        # Results returned by JPL Horizons web interface, using DE432.
+        # TODO: update to de440s, by re-getting the positions.
```

Comment:
This is in a test and is not blocking a merge? Or do you want to do this before merging?

Reply:
I'd really want the tests to remain self-consistent, so a bit of a blocker in that sense. I'd also like to avoid downloading multiple JPL ephemerides every time!

Reply:
I checked JPL horizons for these positions using DE441 (the latest default) and there is no change from DE432, so these positions are probably fine to use the against the DE440s ephemeris for this test.